source/regex.tex

```diff
@@ -869,7 +869,7 @@
 \tcode{error_stack}
 &
 There was insufficient memory to determine whether the regular
```

Comment:
Please convert all the entries in this table into the present tense. (This is another one of ISO/CS's requests, and in this case I think it's reasonable.) That'd also let us replace "would match" with "matches" here without losing the hypothetical feeling.

Reply:
fixed